pkgs/development/ocaml-modules/fileutils/default.nix

```
@@ -10,11 +10,11 @@
 
 buildDunePackage rec {
   pname = "fileutils";
-  version = "0.6.4";
+  version = "0.6.6";
 
   src = fetchurl {
     url = "https://github.com/gildor478/ocaml-fileutils/releases/download/v${version}/fileutils-${version}.tbz";
-    hash = "sha256-enu2vGo2tuvawrTkap6bENNmxaLUQXpfHWih+7oKRF8=";
+    hash = "sha256-eW1XkeK/ezv/IAz1BXp6GHhDnrzXTtDxCIz4Z1bVK+Y=";
   };
 
   minimalOCamlVersion = "4.03";
```

```suggestion
  minimalOCamlVersion = "4.14";
```

https://github.com/gildor478/ocaml-fileutils/compare/v0.6.4...v0.6.6